src/otx/recipe/rotated_detection/openvino_model.yaml

```diff
@@ -1,5 +1,5 @@
 model:
-  class_path: otx.core.model.instance_segmentation.OVRotatedDetectionModel
+  class_path: otx.core.model.instance_segmentation.OVInstanceSegmentationModel
```

Comment:
So the plan is to do rotated detection using instance segmentation models?

Reply:
It is right now, rotated detection is the instance segmentation algorithm with post-processing on the predict stage. I've just removed unnecessary separate file and moved RotatedMaskRCNN to MaskRCNN file. OVRotatedDetectionModel == OVInstanceSegmentationModel